### PR TITLE
Report for challenge #92

### PR DESCRIPTION
**Pull request by pheyvaer**

### CODE REVIEW

File: reports/menubar-app-to-check-who-is-on-vacation.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+   npm start
+   ```
+8. The app appears in the menubar. The icon is an umbrella on a beach.
+
+### Postconditions
+
+- When you click on the icon,
+it shows the vacation days.
+
+## Follow-up actions
```
Comment (bjdmeest) [inline reports/menubar-app-to-check-who-is-on-vacation.md:80]: Who will do them by when?

File: reports/menubar-app-to-check-who-is-on-vacation.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+## Problem
+<!--
+You can reuse the pitch of the challenge, but check if you need to make changes.
+For example, it might happen that the approved solution does more than what the original pitch requested.
+-->
+
+Manually checking your calendar to quickly see who is on vacation is a hassle. 
+It would be easier to have menubar app that shows in a glance who is on vacation today.
+
+## Approved solution
```
Comment (bjdmeest) [inline reports/menubar-app-to-check-who-is-on-vacation.md:14]: I would expect a description of the technical choices, either here or in the readme of the app

**Review by bjdmeest — CHANGES_REQUESTED** [review]
two clarifications

File: reports/menubar-app-to-check-who-is-on-vacation.md (hunk 2 above)
Comment (pheyvaer) [reply reports/menubar-app-to-check-who-is-on-vacation.md:14]: What technical choices do you mean here? Programming language? Frameworks?

File: reports/menubar-app-to-check-who-is-on-vacation.md (hunk 1 above)
Comment (bjdmeest) [reply reports/menubar-app-to-check-who-is-on-vacation.md:80]: It's not clear to me who is responsible to make sure that the follow-up actions are acted upon, but if that's just how it is, then that's a different issue and not relevant for this review.

File: reports/menubar-app-to-check-who-is-on-vacation.md (hunk 2 above)
Comment (pheyvaer) [reply reports/menubar-app-to-check-who-is-on-vacation.md:14]: Added at https://github.com/SolidLabResearch/Challenges/pull/108/commits/8a208ff843c9d4ed66cea2f78afe3643b9f42481. Is this what you expect?

File: reports/menubar-app-to-check-who-is-on-vacation.md (hunk 1 above)
Comment (pheyvaer) [reply reports/menubar-app-to-check-who-is-on-vacation.md:94]: Yeah, that's out-of-scope of the report.

File: reports/menubar-app-to-check-who-is-on-vacation.md (hunk 2 above)
Comment (bjdmeest) [reply reports/menubar-app-to-check-who-is-on-vacation.md:14]: Oh yes perfect!

**Review by bjdmeest — APPROVED** [review]

**Pull request merged**